src/redux/books/apiBooks.js

```diff
@@ -0,0 +1,49 @@
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+const UPDATE = 'bookstore/books/UPDATE';
+const GET = 'bookstore/books/GET';
+const ASYNC_ADD = 'bookstore/books/ASYNC_ADD';
+const ASYNC_REMOVE = 'bookstore/books/ASYNC_REMOVE';
+
+const addBook = (book) => ({
+  type: ADD,
+  book,
+});
+
+const removeBook = (book) => ({
+  type: REMOVE,
+  book,
+});
+
+const updateBook = (book) => ({
+  type: UPDATE,
+  book,
+});
+
+const bookReducer = (state = [], action = {}) => {
+  switch (action.type) {
+    case ADD:
+      if (state.find((elem) => elem.item_id === action.book.item_id)) {
+        return state;
+      }
```

Comment:
- [ ] Kindly remove this if statement because this is unnecessary at this point and can break your redux 🤔